components/infobox/commons/infobox_basic.lua

```diff
@@ -31,4 +34,49 @@ function BasicInfobox:createBottomContent(infobox)
     return nil
 end
 
+--- Allows for using this for customCells
+function BasicInfobox:getMultiArgsForType(args, argType)
+	local typeArgs = {}
```

Comment:
Rename local vars in here to match new function name, etc. I would suggest `foundArgs`